Jenkinsfile.yaml

```diff
@@ -0,0 +1,40 @@
+gitCredentialId: dummyGitRepo
+artifactPublishingConfig:
```

Comment:
Not sure what is this, seems to be unused

Reply:
I was thinking of using this for push operation and to comment on the PR if needed.